### PR TITLE
Prow workloads control plane

### PR DESCRIPTION
**Pull request by fgimenez**
Adds kubespray code to deploy the prow-workloads cluster control plane. Also includes a prow job to trigger it. The ssh key, inventory and vars files required by kubespray will come from the secrets file, this PR puts them in place https://github.com/kubevirt/secrets/pull/15

/cc @dhiller @rmohr

### CODE REVIEW

**Review by dhiller — APPROVED** [review]
/approve
/hold

need to wait for the secrets PR to get merged.

File: github/ci/prow-workloads/hack/deploy.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+        cd ${PROJECT_PATH}
+
+        base_dir=github/ci/prow-workloads
+
+        source ./hack/manage-secrets.sh
+        decrypt_secrets
+        # ssh key
+        mkdir -p ~/.ssh && chmod 0700 ~/.ssh
+        extract_secret 'prowWorkloads.ssKey' ~/.ssh/id_ed25519
```
Comment (dhiller) [inline github/ci/prow-workloads/hack/deploy.sh:15]:
```suggestion
        extract_secret 'prowWorkloads.sshKey' ~/.ssh/id_ed25519
```

**Review by dhiller — CHANGES_REQUESTED** [review]
/lgtm cancel 

one secret key is wrong, see comment.

**Comment by dhiller** [comment]
/hold cancel

Other PR for secrets is merged

**Comment by fgimenez** [comment]
> one secret key is wrong, see comment.

great catch, thanks a lot! :)

**Review by dhiller — APPROVED** [review]

**Pull request merged**